src/renderers/WebGLRenderer.js

```diff
@@ -2012,8 +2013,15 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 		}
 
+		if ( material.envMap instanceof THREE.WebGLRenderTargetCube ) {
+
+			console.warn( 'THREE.WebGLRenderTargetCube: don\'t use render targets as textures. Use their .texture property instead.' );
+			material.envMap = material.envMap.texture;
+
+		}
+
 		uniforms.envMap.value = material.envMap;
-		uniforms.flipEnvMap.value = ( material.envMap instanceof THREE.WebGLRenderTargetCube ) ? 1 : - 1;
+		uniforms.flipEnvMap.value = ( material.envMap && material.envMap.renderTargetCube ) ? 1 : - 1;
```

Comment:
Shouldn't this be `&& material.envMap instanceof THREE.CubeTexture`? I wouldn't add `renderTargetCube` to `Texture` dynamically...